### ISSUE TITLE
Add unique <title> tag to each page

### ISSUE THREAD

**Issue by oneezy** [PolymerElements/polymer-starter-kit#946]
In the Polymer Shop app, a unique `<title>` tag on each page.

I think it would be beneficial to add the same functionality to the PSK.

**Comment by jsilvermist** [comment]
Agreed, I did this in PSK 1, but using a slightly hacky method since without using a `setTimeout` call in the page.js `page` callback it would change the title before calling `pushState`.

**Comment by oneezy** [comment]
Here's the commit that was added in the Shop app for this,
https://github.com/Polymer/shop/pull/32/files
